src/test/java/junit/CalculatorSubTest.java

```diff
@@ -0,0 +1,31 @@
+package junit;
+
+import com.epam.tat.module4.Calculator;
+import org.junit.*;
+import java.util.logging.Logger;
+
+public class CalculatorSubTest {
+    private Calculator calculator;
+    private double firstNumber;
+    private double secondNumber;
+    Logger logger = Logger.getGlobal();
+
+    @Before
+    public void initTest() {
+        calculator = new Calculator();
+        firstNumber = 6.8;
+        secondNumber = 3.3;
+    }
+
+    @Test
+    public void calculateSub() {
+        double actualResult = calculator.sub(firstNumber, secondNumber);
+        Assert.assertEquals("Invalid result of sub operation.", 3.5, actualResult, 0);
```

Comment:
what is 0 here?

Reply:
![Delta](https://user-images.githubusercontent.com/81090703/117199434-db6d5b00-adf2-11eb-908a-fc13a6e99e24.png)
 0 is delta, I have corrected it to 0.0.
 I choose that method because other suitable for me for double type parameters is deprecated and not recommended by IDE: Assert.assertEquals("Invalid result of sum operation.", 10.0, actualResult);
![image](https://user-images.githubusercontent.com/81090703/117199202-9517fc00-adf2-11eb-9b1f-b1bdc9b9d341.png)
Should I ignore IDEs suggestion and warning like that?

Reply:
If method is deprecated - we try not to use it.

Reply:
Thank you for the clarifying.